Vagrantfile

```diff
@@ -0,0 +1,168 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure(2) do |config|
+
+    # NOTE: Ordering matters! The commissaire box should be the
+    #       the last box to start!
+
+    # Development servers server.
+    config.vm.define "servers" do |servers|
+      servers.vm.box = "fedora/24-cloud-base"
+      servers.vm.network "private_network", ip: "192.168.152.101"
+      servers.vm.provision "shell", inline: <<-SHELL
+        echo "==> Setting hostname"
+        sudo hostnamectl set-hostname servers
+        echo "===> Updating the system"
+        sudo dnf update --setopt=tsflags=nodocs -y
+        echo "===> Installing etcd and redis"
+        sudo dnf install -y etcd redis
+        echo "===> Configuring etcd"
+        sudo sed -i "s/localhost/192.168.152.101/g" /etc/etcd/etcd.conf
+        echo "===> Starting etcd"
+        sudo systemctl enable etcd
+        sudo systemctl start etcd
+        echo "===> Set flannel network"
+        sudo etcdctl --endpoint=http://192.168.152.101:2379 set '/atomic01/network/config' '{"Network": "172.16.0.0/12", "SubnetLen": 24, "Backend": {"Type": "vxlan"}}'
+        echo "===> Configure redis"
+        sudo sed -i "s/127.0.0.1/192.168.152.101/g" /etc/redis.conf
+        sudo systemctl enable redis
+        sudo systemctl start redis
+      SHELL
+    # End servers
+    end
+
+    # Development Kubernetes server.
+    # NOTE: This must start after etcd.
+    config.vm.define "kubernetes", autostart: false do |kubernetes|
+      kubernetes.vm.box = "fedora/24-cloud-base"
+      kubernetes.vm.network "private_network", ip: "192.168.152.102"
+      kubernetes.vm.provision "shell", inline: <<-SHELL
+        echo "==> Setting hostname"
+        sudo hostnamectl set-hostname kubernetes
+        echo "===> Updating the system"
+        #sudo dnf update --setopt=tsflags=nodocs -y
```

Comment:
I had commented it out while testing as it was taking a long time to update. Will fix.